src/custom/dataloader/management/commands/etl_features_to_es.py

```diff
@@ -89,3 +92,51 @@ def clean_index(self, index):
             es.indices.delete(index=index, ignore=[400, 404])
         except ElasticsearchException:
             pass
+
+    # Create ES index with specified type mapping from layer source
+    # If mapping not available, we switch on ES type guessing
+    def create_index(self, layer):
```

Comment:
I think it's a good practice to write down the docstring under the definition like so :
```python
    def create_index(self, layer):
        """
        Create ES index with specified type mapping from layer source
        If mapping not available, we switch on ES type guessing
        """
```

Reply:
Indeed, at start it wasn't a function and I forget do migrate the comment ;-) Thanks. 
Fixed !